### DISCUSSION TITLE
加过水印的图片，处理过之后，比如剪裁后无法提取水印

### DISCUSSION THREAD

**Discussion by dudupipi1989** [guofei9987/blind_watermark#68] [Q&A]
加过水印的图片，剪裁后，不能提取出水印了，是乱码，请教一下这怎么解决

**Comment by tdc6597135** [comment]
我也遇到一样的问题  需要对截图做怎样的处理吗？

**Comment by yu1ec** [comment]
> 我也遇到一样的问题 需要对截图做怎样的处理吗？
我也是这样，按理说，需要对截图做什么处理那这个功能就有问题了。
